### ISSUE TITLE
The same image_dir for target_data_config and eval_data_config

### ISSUE THREAD

**Issue by lauraset** [Junjue-Wang/LoveDA#29]
Hello, @Junjue-Wang , I notice that in the config file, the image_dir is the same  for target_data_config  and eval_data_config.  I think the image_dir for target_data_config  should be the training set of target images, while that for eval_data_config should be the validation set of target images. What's your idea? 
![image](https://user-images.githubusercontent.com/39206462/148682300-11837546-3f32-40c3-ad54-5e321e709cf6.png)

**Comment by Junjue-Wang** [comment]
I used the target data config to generate pseudo-labels. The gts of target data are not accessible during the training.
